_posts/2020-09-24-the-business-sense-behind-funcional-programming.md

```diff
@@ -0,0 +1,74 @@
+The FP paradigm has proven[^7] [^8] [^9] to be a much better fit for scalable software. It is, therefore, safe to state: FP, improved long-term growth.
+
+## Human factor
+
+In the software world, managers bemoan the difficulty of hiring FP developers. This difficulty is an undeniable fact. The keen manager however, also recognizes that this difficulty is just another trade-off. Let's take a closer look at the stakes: FP developers are aware of the comparative scarcity of FP jobs. FP developers are perfectly capable of delivering OO software. The typical FP developer *foregoes safe, abundant, high-paying OO jobs* so that they may develop in their favorite paradigm. Any manager who considers FP may therefore expect to run a highly motivated and highly performing team. Hardly the kind which the drive-by headhunter will cripple during the most critical phase of development. You are safe to infer: FP, tight and loyal team.
+
+## Myth-busting
+
+FP adoption has been hindered by a number of misconceptions. Let's set the record straight!
+
+### There are no libraries!
+It depends on the language and the runtime. The most popular FP languages today run on the JVM and feature seamless Java interoperability. In other words: you may use any existing Java library you wish with them.
+
+### I don't trust the runtime!
+If the runtime is a concern, you have a wide selection of languages to choose from which are designed for battle-tested runtimes. Scala and Clojure run on the JVM, F# runs on .NET, Erlang and Elixir run on BEAM. All of these have proven their worth in production under extreme workloads[^10] [^11] [^12] and are actively maintained by large, reliable corporations.
+
+### I can't reuse!
+Whether it's engineering skill or some trusted piece of in-house code, migrating to FP doesn't mean throwing the baby out with the bathwater. You may pick a language which interoperates with the code or skill you want to reuse. Scala and Clojure interoperate with Java. F# interoperates with C#. ClojureScript interoperates with JavaScript and the list goes on.
+
+### It's slow!
+In today's world of distributed computing there is no excuse for complaining about performance without taking parallelism and scalability into account. If you develop performance-sensitive software for single-threaded execution, then FP might not be right tool for you.
+
+### It's only good for mathematicians and academics!
+The many and varied success stories of FP adoption by industry disprove this. Be it networking[^13], messaging services[^14], finance[^15], insurance[^16] or web development[^17], FP languages are proving versatile and capable tools.
```

Comment:
Here it should be either "FP languages are proving to be..." or "FP languages have proved to be..." I think?